### PR TITLE
Prep for 0.11 release

### PR DESCRIPTION
**Pull request by Susurrus**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [package]
 name        = "nix"
 description = "Rust friendly bindings to *nix APIs"
-version     = "0.11.0-pre"
+version     = "0.11"
```
Comment (asomers) [inline Cargo.toml]: Why no trailing ".0" ?

Comment (Susurrus) [reply Cargo.toml]: Good point, fixed.

**Comment by Susurrus** [comment]
Interesting that Cargo won't successfully parse a manifest file with only an X.Y version number.

**Comment by Susurrus** [comment]
We're going to block on this until I sort out the tests I stole in #907.

**Pull request merged**